api/controllers/MerchStoreController.ts

```diff
@@ -190,16 +190,18 @@ export class MerchStoreController {
     return { error: null, order };
   }
 
+  @Get('/orders/all')
+  async getAllMerchOrders(@AuthenticatedUser() user: UserModel): Promise<GetMerchOrdersResponse> {
+    if (!(PermissionsService.canAccessMerchStore(user)
```

Comment:
nit: can we rename `getAllMerchOrders` -> `getMerchOrdersForAllUsers` and `getUserMerchOrders` to `getMerchOrdersForCurrentUser`? Just to make it clearer the difference (ie. we can't do "get all orders for any user, only the one who is making the call")